src/Campaigns/resources/admin/components/CampaignDetailsPage/index.tsx

```diff
@@ -120,29 +199,17 @@ export default function CampaignsDetailsPage({campaignId}) {
                         <div className={styles.flexContainer}>
                             <div className={styles.flexRow}>
                                 <h1 className={styles.pageTitle}>{campaign.title}</h1>
-                                <StatusBadge />
+                                <span
+                                    className={cx(
+                                        styles.status,
+                                        styles[`${campaign.status}Status`],
+                                    )}
+                                >
+                                    {getStatus(campaign.status)}
```

Comment:
Is there some specific reason to replace the `<StatusBadge />` component with it? 

The main advantage of using the previous component is that it uses the same status styles from the list tables that live on this file: https://github.com/impress-org/givewp/blob/epic/campaigns/src/Views/Components/ListTable/InterweaveSSR/styles.scss

Reply:
That was a result of resolving the merge conflict 😅 
Resolved: https://github.com/impress-org/givewp/pull/7565/commits/38f5fe7165024960ff6f09c973aff2fb4b68f406